components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FxaClient.kt

```diff
@@ -115,7 +115,7 @@ class FxaClient(inner: FirefoxAccount, persistCallback: PersistCallback?) : Auto
         entrypoint: String,
```

Comment:
We should expose the service param here and use the default value from here since there is this thin wrapper around the rust generated FFI code.

```suggestion
        entrypoint: String,
        services: List<String> = listOf("sync"),
```